src/lib.rs

```diff
@@ -68,10 +72,160 @@ mod connect {
     where
         S: AsyncRead + AsyncWrite + Unpin,
     {
-        let builder = native_tls::TlsConnector::builder();
-        let connector = builder.build()?;
-        let connector = TlsConnector::from(connector);
-        let stream = connector.connect(domain, stream).await?;
+        let stream = TlsConnector::new().connect(domain, stream).await?;
         Ok(stream)
     }
+
+    /// Connect a client to a remote server.
+    ///
+    /// # Examples
+    ///
+    /// ```no_run
```

Comment:
lets run these please, so I am sure it acutally works

Reply:
done!